codes/codes.go

```diff
@@ -25,7 +25,13 @@ import (
 	"strconv"
 )
 
-// A Code is an unsigned 32-bit error code as defined in the gRPC spec.
+// A Code is a status code according to the gRPC spec:
+//
+// https://github.com/grpc/grpc/blob/master/doc/statuscodes.md
+//
+// Only the codes defined as consts in this package are valid codes. Do not use
+// other code values.  Behavior and interopability are implementation-specific
+// and not guaranteed.
```

Comment:
Can you clarify the scope of the language of "behavior and interoperability" here? What is not guaranteed? Any permutation of languages with defined status codes? Only status codes undefined?

Reply:
Clarified.